--- pypdfocr_watcher.py	2016-09-20 10:02:06.556301294 +0000
+++ pypdfocr_watcher.py	2016-09-20 10:04:21.092354886 +0000
@@ -93,7 +93,7 @@

         """
         if ev_path.endswith(".pdf"):
-            if not ev_path.endswith("_ocr.pdf"):
+            if not ev_path.endswith(("_ocr.pdf","_text.pdf")):
                 PyPdfWatcher.events_lock.acquire()
                 if not ev_path in PyPdfWatcher.events:
                     PyPdfWatcher.events[ev_path] = time.time()